xclim/indices/stats.py

```diff
@@ -1,14 +1,19 @@
 """Statistic-related functions. See the `frequency_analysis` notebook for examples."""
 from __future__ import annotations
 
+import warnings
 from typing import Any, Sequence
 
+import flox.xarray as floxx
```

Comment:
`flox` is currently listed as an optional dependency (it's not even listed as a dev dependency). Is there a way to make this import optional as well or are we officially adopting this here?

Reply:
I didn't use flox in the end, didn't seem worth it

Reply:
If there is a significant speedup that you think we could benefit from, feel free to add it in. Otherwise, what I would propose is opening an issue to discuss adopting `flox` as a "true" optional dependency (like `lmoments3`) and seeing where we could easily integrate it for analysis speedups.